src/Settings/JSONSettings.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+namespace JSONSettings
+{
+	class SettingsHolder
+	{
+	public:
+		static SettingsHolder* GetSingleton()
+		{
+			static SettingsHolder singleton;
+			return std::addressof(singleton);
+		}
+
+		void Read();
+		[[nodiscard]] const bool IsProhibitedSpell(const RE::SpellItem* a_spell);
```

Comment:
const should be at the trailing end of the function. returning a const bool doesn't actually change the signature.